### ISSUE TITLE
Microsoft.WinAppDriver.Appium.WebDriver is Preview (prerelease) since 18/05/2018 - Is development stopped on this ?

### ISSUE THREAD

**Issue by vikramksrm** [microsoft/WinAppDriver#1281]
Firstly this is fantastic framework! Kudos to the team. Can I go ahead and use this?

**Comment by anunay1** [comment]
There is no support from Microsoft, although they have made this an open source but they have not shared any source, and there is no plan shared MS on this. 

You can check the below repo: 
https://github.com/kfrajtak/WinAppDriver

he has shared the source also.

**Comment by vikramksrm** [comment]
> There is no support from Microsoft, although they have made this an open source but they have not shared any source, and there is no plan shared MS on this.
> 
> You can check the below repo:
> https://github.com/kfrajtak/WinAppDriver
> 
> he has shared the source also.
> 
> I would suggest that you stay away from this.

Thnk you for reply, but kfrajtak/WinAppDriver this doesnt have any releases!

**Comment by anunay1** [comment]
Yes it does not have release you can just clone the repo build it and start the server
